common/changes/@rushstack/eslint-plugin-security/will-issue-547_2022-07-01-17-58.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/eslint-plugin-security",
+      "comment": "Rename cyclicDependencyProjects to decoupledLocalDependencies",
+      "type": "minor"
```

```suggestion
      "comment": "",
      "type": "none"
```

This change doesn't affect consumers of the package.